app/src/main/java/pl/example/android/garbageapp/utils/NotificationUtils.java

```diff
@@ -1,30 +1,96 @@
-                        .setContentTitle("Informacja")
-                        .setContentText(contentText);
+    public static void showNotification(Context context, int notificationId, int notificationSectorColor) {
         NotificationManager notificationManager =
                 (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            NotificationChannel notificationChannel = new NotificationChannel(
+                    GARBAGE_APP_NOTIFICATION_CHANNEL_ID,
+                    context.getString(R.string.main_notification_channel_name),
+                    NotificationManager.IMPORTANCE_DEFAULT
+            );
+            notificationManager.createNotificationChannel(notificationChannel);
+        }
+        NotificationCompat.Builder notificationBuilder =
+                new NotificationCompat.Builder(context, GARBAGE_APP_NOTIFICATION_CHANNEL_ID)
+                        .setColor(ContextCompat.getColor(context, R.color.colorPrimary))
+                        .setSmallIcon(R.mipmap.ic_delete_sweep_black_24dp)
+                        .setLargeIcon(largeIcon(context))
+                        .setContentTitle(contentTitle(context))
+                        .setContentText(contentText(context))
+                        .setContentIntent(contentIntent(context, notificationSectorColor))
+                        .setDefaults(Notification.DEFAULT_ALL)
+                        .setAutoCancel(true);
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN
+                && Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
+            notificationBuilder.setPriority(NotificationCompat.PRIORITY_HIGH);
+        }
         notificationManager.notify(notificationId, notificationBuilder.build());
     }
+
+    private static PendingIntent contentIntent(Context context, int notificationSectorColor) {
+        Intent startActivityIntent = new Intent(context, whichActivityToStart(notificationSectorColor));
+        startActivityIntent.setFlags(Intent.FLAG_ACTIVITY_SINGLE_TOP);
+        return PendingIntent.getActivity(
+                context,
+                GARBAGE_APP_PENDING_INTENT_ID,
+                startActivityIntent,
+                PendingIntent.FLAG_UPDATE_CURRENT);
+    }
+
+    private static CharSequence contentTitle(Context context) {
+        return context.getString(R.string.notification_message_title);
+    }
+
+    private static CharSequence contentText(Context context) {
+        return context.getString(R.string.notification_message_content);
+    }
+
+    private static Bitmap largeIcon(Context context) {
+        Resources res = context.getResources();
+        Bitmap largeIcon = BitmapFactory.decodeResource(res, R.mipmap.ic_launcher_round);
+        return largeIcon;
+    }
+
+    private static Class whichActivityToStart(int notificationSectorColor) {
+        switch (notificationSectorColor) {
```

Comment:
Please use enumeration values BLUE, GREEN, YELLOW.  Use SectorColor valueOf(int type) to convert int to SectorColor.